docs/faq.md

```diff
@@ -42,6 +42,19 @@ https://docs.docker.com/desktop/
 **A:**
 -->
 
+<!-- #640 -->
+**Q: How can I perform a clean uninstall of Rancher Desktop?**
+
+**A:** The uninstall process varies based on the operating system. For more information, please refer [here](https://docs.rancherdesktop.io/installation).
```

Comment:
Doing factory reset would still help in a cleaner uninstall than uninstall alone.  Uninstall alone would preserve the user  preferences from the previous install.

So, I would rephrase the answer to:

First perform a `Factory reset` (you can link to factory reset  section here) and then uninstall the app. The uninstall process varies based on the operating system. For more information, please refer [here](https://docs.rancherdesktop.io/installation).

Reply:
Revised per feedback, ty!